tests/utils.go

```diff
@@ -4281,6 +4288,16 @@ func HasCDI() bool {
 	return HasDataVolumeCRD()
 }
 
+func isARM64() bool {
+	return rt.GOARCH == "arm64"
```

Comment:
Oh, I just realize that now, but you are actually checking if the machine where you run the functest binary is `arm64` I think you have to check the cluster on which architecture it runs (you should be able to just look it up from the node objects).

Reply:
Yes, you are right, sorry about the thoughtless.